mu-plugins/blocks/screenshot-preview/src/screenshot.js

```diff
@@ -69,6 +69,7 @@ function ScreenShotImg( { src, isReady = false } ) {
 	const convertResponseToBase64 = async ( res ) => {
 		const blob = await res.blob();
 
+		/* eslint-disable no-undef */
```

Comment:
Is `FileReader` a global? If you add it to the top of the file, ex `/* global FileReader */`, eslint should understand that. Disabling `no-undef` like this also disables it for the rest of the file.

Reply:
Ah, good catch. I wanted `// eslint-disable-next-line`. 

Adding it as a global to the top of the file works too.